houndigrade/cli.py

```diff
@@ -309,6 +324,8 @@ def mount(partition, inspect_path):
         inspect_path (str): The path where the partition should be mounted.
 
     """
+    global INSPECT_PATH
```

Comment:
Manipulating globals always feels _awkward_ and a little worrisome to me, but I understand what you're doing here, and I think it's fine here since houndigrade's general footprint is so small.

This isn't a blocker; just throwing a comment here to solicit any other comments or ideas you may have had about that. Feel free to resolve and proceed if not. 🙂

Reply:
Left my response in the slack thread: https://ansible.slack.com/archives/C8VGAPJNN/p1649183796289639?thread_ts=1649165267.080299&cid=C8VGAPJNN